test/suite/test_timestamp03.py

```diff
@@ -254,16 +334,56 @@ def test_timestamp03(self):
             count += 1
         # print "Updated " + str(count) + " keys to value3"
 
+        # Scenario : 6
+        # when read time stamp != stable timestamp, take checkpoints using both
+        # stable timestamp and read timestamp and check to see the values differ
+        # and are proper.
+        if self.ckptcfg == 'use_timestamp=true':
+            # make sure stable_timestamp is set.
+            ckpt_ts = 'stable_timestamp=' + timestamp_str(100 + nkeys)
+            self.conn.set_timestamp('stable_timestamp=' + timestamp_str(100 + nkeys))
+            valcnt2 = valcnt3 = valcnt4 = nkeys
+            valcnt = 0
+            self.ckpt_backup(self.value3, valcnt, valcnt2, valcnt3, valcnt4, "")
+            # checkpoint at latest timestamp with read_timestamp
+            ckpt_ts = 'read_timestamp=' + timestamp_str(200 + nkeys)
+            valcnt = nkeys
+            self.ckpt_backup(self.value3, valcnt, valcnt2, valcnt3, valcnt4, ckpt_ts)
+
+        # If we're not using the log we're done.
+        if not self.using_log:
+            return
+
+        # Update them and retry.  This time take a backup and recover.
```

Comment:
please be more specific about what "Update them and retry" means.  What is 'them'?